akka-projection-core/src/main/scala/akka/projection/ProjectionSettings.scala

```diff
@@ -55,16 +58,22 @@ object ProjectionSettings {
   def create(system: ClassicActorSystemProvider): ProjectionSettings = apply(system)
 
   def apply(system: ClassicActorSystemProvider): ProjectionSettings = {
-    fromConfig(system.classicSystem.settings.config.getConfig("akka.projection.restart-backoff"))
+    fromConfig(system.classicSystem.settings.config.getConfig("akka.projection"))
   }
 
-  def fromConfig(config: Config) =
+  def fromConfig(config: Config) = {
+    val restartBackoffConfig = config.getConfig("restart-backoff")
+    val atLeastOnceConfig = config.getConfig("at-least-once")
+    val recoveryStrategyConfig = config.getConfig("recovery-strategy")
```

Comment:
Maybe these should all be split into their own setting types and implementations.